### PR TITLE
Update docs to include Sparse Fieldset information

### PR DESCRIPTION
**Pull request by ghunti**

### CODE REVIEW

**Comment by ghunti** [comment]
I wasn't sure how to build the docs to see if everything was ok, so there might be some formatting issues.
If you point me to some docs on how to build the docs locally I can give it a look

**Comment by matthewtrask** [comment]
I think the docs are Jekyll, so you need to have that locally on your machine. I will pull this down tonight, and see that it builds ok. Thanks for this!

**Comment by matthewtrask** [comment]
@ghunti can you update the PR to point to our gh-pages in the repo, not the leagues? I think that would be the right target branch. Otherwise it looks good.

**Comment by ghunti** [comment]
@matthewtrask sorry for the delay. I don't understand your last comment, can you help me? :-D

File: transformers.md [hunk 1]
```diff
@@ -307,3 +307,48 @@ huge number of SQL requests.
...
+        return $this->item($author, new AuthorTransformer, 'people');
+    }
+    ...
+~~~
+
+With this, you can now filter the fields returned by the API using `/books?fields[book]=title,year`.
+
+If you want to filter out relation fields that were included in the request, you need to specify the relation as an attribute of the resource [as defined by the spec](https://jsonapi.org/examples/#sparse-fieldsets).
+
+**E.g:** `/books?included=authorf&ields[book]=title,year,author&fields[people]=name`
```
Comment (JSteitz) [inline transformers.md:354]: Typo: /books?included=**authorf&ields**[book]=title,year,author&fields[people]=name

**Pull request closed without merging**